giraffe/src/components/Axes.tsx

```diff
@@ -100,9 +100,19 @@ export const drawAxes = ({
   context.textAlign = 'end'
   context.textBaseline = 'middle'
   const yDomainWidth = yDomain[1] - yDomain[0]
+  if (yColumnLabels) {
+    let y
+    for (let i = 0; i < yColumnLabels.length; i++) {
+      y = yScale(i) + margins.top - height / (yColumnLabels.length * 2)
 
-  if (yColumnType === 'string') {
-    //TODO: implement
+      context.globalAlpha = 1
+      context.fillStyle = tickFontColor
```

Comment:
I understand why you added these here, but they can be moved up above the loop.

canvas is a state machine. What that means is, you set the state, and it retains whatever state you set until you change the state by setting it to something else. Inside of a loop with no conditions (i.e. there's no `if` statements in our loop), you can move it to outside of the loop, since every item in the loop will use the same style.